clients/privacy-center/pages/index.tsx

```diff
@@ -130,20 +130,39 @@ const Home: NextPage = () => {
             >
               {config.title}
             </Heading>
-            <Text
-              fontSize={["small", "medium"]}
-              fontWeight="medium"
-              maxWidth={624}
-              textAlign="center"
-              color="gray.600"
-              data-testid="description"
-            >
-              {config.description}
-            </Text>
+
+            {config.description.map((paragraph) => {
```

Comment:
This isn't backwards compatible though - it'll break existing configs that are a `string` and not an array